src/openforms/authentication/contrib/generic_oidc/backends.py

```diff
@@ -0,0 +1,194 @@
+        return False
+
+    def get_or_create_user(
+        self, access_token: str, id_token: str, payload: JSONObject
+    ) -> AnonymousUser:
+        """
+        Return a "fake" Django user.
+
+        If the claims are valid, we only process them and do not create or update an
+        actual Django user.
+        """
+        assert isinstance(self.request, HttpRequest)
+
+        user_info = self.get_userinfo(access_token, id_token, payload)
+        if not self.verify_claims(user_info):
+            # Raise PermissionDenied rather than SuspiciousOperation - this makes it
+            # Django stops trying other (OIDC) authentication backends, which fail
+            # because the code was already exchanged for an access token.
+            # Note that this backend only runs for the OF OIDC configs at all,
+            # and those aren't particularly compatible with the admin-OIDC flow anyway.
+            # See :meth:`_check_candidate_backend` that prevents this backend from being
+            # used for admin OIDC.
+            raise PermissionDenied("Claims verification failed")
+
+        self._extract_and_store_claims(payload)
+
+        user = AnonymousUser()
+        user.is_active = True  # type: ignore
+        return user
+
+    def _process_claims(self, claims: JSONObject) -> JSONObject:
+        # see if we can use a cached config instance from the settings configuration
+        assert hasattr(self, "_config") and (
+            isinstance(self._config, BaseConfig)
+            or isinstance(self._config, YiviOpenIDConnectConfig)
+        )
+
+        # Because Yivi has loa config for BSN and KVK, we need to set the "global" loa
+        # config dynamically based on which data is presented.
+        # This will be moved after oidc-db changes.
+        if issubclass(self.config_class, YiviOpenIDConnectConfig):
+            # Check if the claims contain the claim for BSN.
+            # If so then the authentication is for BSN
+
+            # Set the "global" loa config based on the used authentication method
+            match (
+                glom(claims, Path(*self._config.bsn_claim), default=None),
+                glom(claims, Path(*self._config.kvk_claim), default=None),
+            ):
+                case str(), _:
+                    self._config.loa_claim = self._config.bsn_loa_claim
+                    self._config.default_loa = self._config.bsn_default_loa
+                    self._config.loa_value_mapping = self._config.bsn_loa_value_mapping
+                case None, str():
+                    self._config.loa_claim = self._config.kvk_loa_claim
+                    self._config.default_loa = self._config.kvk_default_loa
+                    self._config.loa_value_mapping = self._config.kvk_loa_value_mapping
+                case None, None:
+                    self._config.loa_claim = [""]
+                    self._config.default_loa = None
+                    self._config.loa_value_mapping = None
```

Comment:
I don't see definitions for the `loa_claim`, `default_loa` and/or `loa_value_mapping` properties/attributes on the `YiviOpenIDConnectConfig` model. My type checker is not happy about this :grimacing: 

![image](https://github.com/user-attachments/assets/09695dda-1e31-4407-8b6b-9a0e000978e4)